packages/flutter_adaptive_scaffold/example/pubspec.yaml

```diff
@@ -12,12 +12,14 @@ dependencies:
     sdk: flutter
   flutter_adaptive_scaffold:
     path: ..
-  flutter_lints: ^2.0.0
+  flutter_lints: ^3.0.1
 
 dev_dependencies:
   build_runner: ^2.1.10
   flutter_test:
     sdk: flutter
+  integration_test:
```

Comment:
This dependency is what's causing the problematic Podfile to be created. We could add the adjusted Podfile, but looking at the PR it's not clear to me why this is being added in the first place. What's the reason for adding it?

Reply:
Looks like it was added in the past because of mistake, but somehow I forgot to remove it.